hack/set-motd/main.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"time"
+)
+
+type stringFlag struct {
```

Comment:
What does "stringFlag" means?

Reply:
But if someone read the struct (only the name) should represent the item/object inside ... I'm just talking about the naming